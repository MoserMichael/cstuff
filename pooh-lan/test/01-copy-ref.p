ff := 42 
gg := ff
println( ~msg ff )
println( ~msg gg )
ff = 43
println( ~msg ff )
println( ~msg gg )

##############################################

aaa = []

aaa[ 1 ] = 42

aaa[ 2 ] := aaa[ 1 ]

println( ~msg aaa[ 1 ] )

println( ~msg aaa[ 2 ] )

aaa[ 1 ] = 43

println( ~msg aaa[ 1 ] )

println( ~msg aaa[ 2 ] )

##############################################

a = 42

println( ~msg a )

b := a

println( ~msg b )

a = 43

println( ~msg a )
println( ~msg b )

##############################################

aa = 'a'

println( ~msg aa )

bb := aa

println( ~msg bb )

aa = 'aa'

println( ~msg aa )
println( ~msg bb )







