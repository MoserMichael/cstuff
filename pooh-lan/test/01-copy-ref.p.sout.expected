42
42
43
43
42
42
43
43
42
42
43
43
a
a
aa
aa
